### ISSUE TITLE
release.py cannot find Tag checkout

### ISSUE THREAD

**Issue by Niedes** [zestsoftware/zest.releaser#213]
```
Traceback (most recent call last):
  File "../../bin/release", line 27, in <module>
    sys.exit(zest.releaser.release.main())
  File "/plone/plone5/eggs/zest.releaser-6.9-py2.7.egg/zest/releaser/release.py", line 335, in main
    releaser.run()
  File "/plone/plone5/eggs/zest.releaser-6.9-py2.7.egg/zest/releaser/baserelease.py", line 353, in run
    self.execute()
  File "/plone/plone5/eggs/zest.releaser-6.9-py2.7.egg/zest/releaser/release.py", line 72, in execute
    self._release()
  File "/plone/plone5/eggs/zest.releaser-6.9-py2.7.egg/zest/releaser/release.py", line 273, in _release
    os.chdir(tagworkingdir)
OSError: [Errno 2] No such file or directory: '/tmp/my.pkg-2.1.dev0-GVvUoM/my.pkg'
```

I had to change 
`if self.vcs.relative_path_in_repo:`
to
`if False:`
to make it work.

I'm in a relative path on svn, my svn structure looks like this:

* svn
  * Project1
    * branches
    * tags
    * trunk
  * Project2
    * branches
    * tags
    * trunk

**Comment by mauritsvanrees** [comment]
Do you *have* to be in a relative path when you make a release? If you are in the root of your package, next to the `setup.py`, you will not enter this code path.

The code path is there for the case when you have checked out `Project1/trunk` and the `setup.py` is at `Project1/trunk/some/directory/setup.py` instead of in `trunk`.

I wonder if you do this in a checkout of `svn` instead of what I expect: a checkout of `svn/Project1/trunk`. If that is the case, then your use case is not supported.  If you have multiple projects that you want to handle in one checkout, then I expect this structure:

* svn
  * trunk
    * Project1
    * Project2
  * branches
  * tags

In that case, `zest.releaser` allows you to have a checkout of `trunk` and do the release in the `trunk/Project1` directory, using the package info from `trunk/Project1/setup.py`, and ultimately creating a tag like this:

* svn
  ...
  * tags
    * 1.0
      * Project1
      * Project2

**Comment by Niedes** [comment]
Ah, I think my use case is not supported then, there is an additionoal directory between trunk and setup.py - it looks like this:

- svn
  - Project1 (some java stuff)
    - branches
    - tags
    - trunk
      - some directories and files
  - Project2 (python packages)
    - branches
    - tags
    - trunk
      - my-pkg1
        - setup.py
        - ...
      - my-pkg2
        - setup.py
        - ...

I check out `Project2/trunk`, there I cd to my-pkg1, there I call `../path/to/fullrelease`.
Answering `Y` to `You are NOT in the root of the repository. Do you want to go there? (Y/n)?` gives
`CRITICAL: No version found.`, answering `n` yields in the error described above.

**Comment by Niedes** [comment]
In the temp directory is another directory called `/tmp/my.pkg-2.1.dev0-GVvUoM/`.
In this directory there are the contents of `my.pkg` (the setup.py, the src folder and so on).
But release.py py expects this content to be in `/tmp/my.pkg-2.1.dev0-GVvUoM/my.pkg/`.

**Comment by mauritsvanrees** [comment]
Fixed in 6.12.2.
